src/apps/api/tests/test_competitions.py

```diff
@@ -0,0 +1,52 @@
+import json
+
+from django.urls import reverse
+from rest_framework.test import APITestCase
+
+from competitions.models import CompetitionParticipant
+from factories import UserFactory, CompetitionFactory, CompetitionParticipantFactory, PhaseFactory, LeaderboardFactory, \
+    ColumnFactory
+
+
+class CompetitionParticipantTests(APITestCase):
+    def setUp(self):
+        self.creator = UserFactory(username='creator', password='creator')
+        self.other_user = UserFactory(username='other_user', password='other')
+        self.comp = CompetitionFactory(created_by=self.creator)
+        PhaseFactory(competition=self.comp)
+        self.leaderboard = LeaderboardFactory(competition=self.comp)
+        ColumnFactory(leaderboard=self.leaderboard)
+
+    def _prepare_competition_data(self, url):
+        # data = CompetitionSerializer(comp).data
```

Comment:
pull

Reply:
✔️ (note, left this in so we could discuss this way vs doing a `GET` to get competition details)